cloud-account/index.md

```diff
@@ -6,13 +6,24 @@ mapped_urls:
 
```

Comment:
please add `applies` to the front matter of these pages and the others in the section

https://elastic.github.io/docs-builder/syntax/applies.html

Reply:
done.